### ISSUE TITLE
TOC macro in use on Vets API on EKS

### ISSUE THREAD

**Issue by BSmolensky** [department-of-veterans-affairs/va.gov-team#79348]
While checking the Vets API on EKS page fro deploying, I noticed that the page included the unsupported TOC macro. The owners of the page should be notified that this functionality will not appear on the PW and give them the option to delete it from the Confluence page. 

### TASKS

- [x] Identify page owner(s)
- [x] Slack them that the macro is not supported. 
- [x] Stress that this is not a LBing issue, just an unsupported macro. 
- [x] Let them know that they can keep the macro (for desired Confluence functionality) or delete it. 
- [x] Be available for follow-up questions if necessary. 

### Definition of Done

- [x] Page owners have been notified and their questions answered.

**Comment by BSmolensky** [comment]
Slacked Lindsey Hattamer and Rachal Cassidy the following: 

>Hi Lindsey and Rachal - During my deployment today, I was reviewing Vets API on EKS and noticed something I wanted to bring to your attention. The Table of Contents Macro -- which currently exists on the page after the intro paragraph -- will NOT appear on the PW. That macro is not supported and I wanted to make you aware of that fact. When that page deploys, there will simply be a somewhat larger than normal space on the page where the TOC Macro lived.
The reason it is not supported is because the PW's theme auto-generates a scrollable TOC that appears on the right hand side of all pages. So  PW users already have the ability to see -- and click on -- the sections of PW pages.
Using the TOC macro is NOT a launch-blocking issue. I simply wanted to let you know that it will not appear on the PW website and that you can delete it if you would like. Sorry to make this such a long slack...but I wanted to make sure I didn't confuse anyone...most of all myself. 
Please let me know if you have any questions.

**Comment by BSmolensky** [comment]
Lindsey Hattamer replied: 
Thanks for the info! (at) Rachal Cassity do you want to go ahead and remove that when you get a chance?

**Comment by BSmolensky** [comment]
Rachal Cassity replied:

> Thank you for letting me know, I’ll definitely keep that in mind. Is there another widget we can use for the TOC?

**Comment by BSmolensky** [comment]
I replied: 

> Sadly, there is not an alternative at this time. Please keep in mind that we -- the Platform Content Team -- have no control over that level of functionality. [Those decisions are made by the team at k15t](https://help.k15t.com/scroll-viewport/why-does-the-table-of-contents-or-page-tree-macro-). Attached is a screen shot of 
the Vets API on EKS page on the PW. The highlighted area is the PW theme's TOC that appears on all PW pages. The k15t squad has disabled TOC macro functionality so as not to duplicate the info on a page.
We understand that the unsupported TOC macro feature offers Confluence users the ability to see everything on the page and to jump down to a section without scrolling. This makes editing and working on pages in Confluence much easier, especially on long docs like Vets API on EKS. Adding the unsupported TOC macro doesn't affect the end product on the PW, it simply isn't there (and there is a tad more space where the TOC should be). If your team desires the "jump down" ability while working on the page in Confluence, you are more then welcome to keep the TOC on your PW-destined pages. I wanted to make sure you and your team are aware that the TOC macro will not appear on the PW.
This is an issue that only recently came to our attention as we began work to optimize the site and improve the UX. We will be making a Tip of the Sprint about it in an attempt to help get the word out. You are not the only team currently using the TOC macro on pages. We are reaching out to teams when we see it on pages during deployment.
Dang. Sorry. This is a long Slack again. Hopefully I have answered your question and helped you to understand what's going on. Or at the very least, made you dislike k15t. I will stop typing now. Have a good night. Okay...stopping the typing for real this time. 
<img width="1440" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN01TQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5b0605817e553ca24dc5876cdf91b218ebd3d97b/Screen%20Shot%202024-03-26%20at%208.48.21%20PM.png" alt="Screen Shot 2024-03-26 at 8.48.21 PM.png" />
